CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
```

Comment:
Remove the Changelog, we only add this type of file before the first release, now we have no changes between two different releases.

Reply:
I thought it could be useful since I am already describing the features introduced with this PR. Indeed, I used the `Unreleased` keyword